config.c

```diff
@@ -277,7 +280,8 @@ void handleConfigSet(RedisRaftCtx *rr, RedisModuleCtx *ctx, RedisModuleString **
     valuebuf[value_len] = '\0';
 
     char errbuf[256] = "ERR ";
-    if (processConfigParam(keybuf, valuebuf, rr->config, false,
+    if (processConfigParam(keybuf, valuebuf, rr->config,
+                           false, rr->state == REDIS_RAFT_UNINITIALIZED,
```

Comment:
Redis thread will call this function and access is not synchronised with raft thread. (e.g rr->state is updated from raft thread). I think, we still need to create RaftReq and handle this change in raft thread.

Reply:
right.  though isn't that in practice true for all set/get config variables?  or do we not care so much about the rest?

Reply:
Yes, true for all. If we move this to raft thread, we are fixing the issue for all I guess. If it requires complex changes, we can create a follow-up PR if you want.